.github/workflows/publish-npm-canary.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
           node --version
           yarn --version
       - name: Install dependencies
-        run: yarn install --frozen-lockfile --check-files
+        run: yarn install --immutable --check-files
```

Comment:
I believe `--check-files` has been deprecated

Reply:
It was removed and should be throwing as it's an unknown flag